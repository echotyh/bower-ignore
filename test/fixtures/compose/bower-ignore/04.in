{
  "in": {
    "dependencies": {
      "bootstrap": "^3.3.6",
      "font-awesome": "fontawesome#^4.6.3",
      "test": "^0.0.1"
    },
    "dependencies-gitignore": {
      "bootstrap": {
        "ignore": ["*"],
        "unignore": ["dist/"]
      },
      "font-awesome": {
        "ignore": ["test1"],
        "unignore": ["test2"]
      }
    }
  },
  "force": true,
  "update": {
    "bootstrap": {
      "ignore": ["test1"],
      "unignore": ["test2/"]
    },
    "test": {
      "ignore": ["test1"],
      "unignore": ["test2"]
    }
  }
}
