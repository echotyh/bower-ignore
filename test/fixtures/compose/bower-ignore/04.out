{
  "bootstrap": {
    "ignore": ["test1"],
    "unignore": ["test2/"]
  },
  "font-awesome": {
    "ignore": [],
    "unignore": []
  },
  "test": {
    "ignore": ["test1"],
    "unignore": ["test2"]
  }
}
